app/src/main/java/com/wireguard/android/fragment/TunnelListFragment.kt

```diff
@@ -5,18 +5,17 @@
  */
 package com.wireguard.android.fragment
 
+import android.app.SearchManager
+import android.content.Context
 import android.content.Intent
 import android.content.res.Resources
 import android.net.Uri
 import android.os.Bundle
 import android.provider.OpenableColumns
-import android.view.LayoutInflater
-import android.view.Menu
-import android.view.MenuItem
-import android.view.View
-import android.view.ViewGroup
+import android.view.*
```

Comment:
I don't think Spotless should have allowed this, did you run `./gradlew spotlessApply`?

Reply:
Oh forgot that, applying right now

Reply:
Tried `./gradlew spotlessApply` and it passes it :sweat_smile:, I can import them manually and remove * if you want.